packages/connect/src/core/onCallFirmwareUpdate.ts

```diff
@@ -456,6 +457,13 @@ export const onCallFirmwareUpdate = async ({
         context: { deviceList, device: reconnectedDevice, log, postMessage },
     });
 
+    // features.firmware_present non-null value implies that device was initially connected with
+    // features.bootloader_mode=true, which means that no automatic language update was performed
+    if (reconnectedDevice.atLeast(['2.7.0']) && deviceInitiallyConnectedWithoutFirmware) {
```

Comment:
Can we pull the feature support from config rather than having the versions hard-coded? There are multiple instances in this file.